### DISCUSSION TITLE
软连接被识别为文件

### DISCUSSION THREAD

**Discussion by mengy822** [1Panel-dev/1Panel#5872] [Q&A]
### 1Panel 版本

v1.10.12-lts

### 问题描述

arm debain系统 最新版本1panel 文件夹的软连接被识别为文件

### 重现步骤

arm debain系统 最新版本1panel 文件夹的软连接被识别为文件

**Comment by wanghe-fit2cloud** [comment]
脚本类型的软链接可以点击编辑，但二进制等类型文件有编辑限制。
<img width="1176" alt="image" src="https://github.com/user-attachments/assets/b99ebcb5-aaa9-4fa6-acb3-ab033d12e218">

**Reply by mengy822** [reply]
> 脚本类型的软链接可以点击编辑，但二进制等类型文件有编辑限制。 <img alt="image" width="1176" src="https://private-user-images.githubusercontent.com/46734380/350327964-b99ebcb5-aaa9-4fa6-acb3-ab033d12e218.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0NDE4NzIsIm5iZiI6MTcyMTQ0MTU3MiwicGF0aCI6Ii80NjczNDM4MC8zNTAzMjc5NjQtYjk5ZWJjYjUtYWFhOS00ZmE2LWFjYjMtYWIwMzNkMTJlMjE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIwVDAyMTI1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2MGUzYzI3OGM2YWU2NDJmNWYzYWU5MjdlN2JlZmZiYWY2NDM2NWUyYzlkZmVmNDZmY2NjMjM3NmQ3ZmEyZTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PH8AYiZdTO2IJRUc9goD28gPEJOZlFPiZ-1HLWoxgPw">

可我的是文件夹的软连接呀

**Reply by wanghe-fit2cloud** [reply]
后面版本优化。

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/issues/5879
